src/niswitch/metadata/enums.py

```diff
@@ -314,7 +314,7 @@
                 'documentation': {
                     'description': 'Resource in use'
                 },
-                'name': 'NISWITCH_VAL_RESOURCE_IN_USE',
+                'name': 'NISWITCH_VAL_RSRC_IN_USE',
```

Comment:
This is a good catch! We need to fix our internal metadata.

Reply:
Changing the enum value would break source compatibility; I'll update the documentation to match the enum value.